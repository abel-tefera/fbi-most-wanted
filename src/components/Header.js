import React from 'react';
import { Outlet } from 'react-router-dom';

const Header = () => (
  <>
    <nav className="flex flex-row justify-between py-12">
      <div>FBI Watch List</div>
    </nav>
    <Outlet />
  </>
);

export default Header;
