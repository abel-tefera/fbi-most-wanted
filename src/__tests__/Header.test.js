import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('tests for header component', () => {
  test('snapshot for header', () => {
    const component = renderer
      .create(
        <BrowserRouter>
          <Header />
        </BrowserRouter>,
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
