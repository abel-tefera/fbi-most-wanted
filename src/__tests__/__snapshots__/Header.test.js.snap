// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for header component snapshot for header 1`] = `
Array [
  <nav
    aria-controls="accordion-arrow-icon-body-1"
    className="flex justify-center w-full bg-custom-blue-dark font-lato items-center"
  >
    <p
      className="text-sm font-semibold italic text-white"
    >
      Fidelity, Bravery, Integrity
    </p>
  </nav>,
  <footer
    className="flex w-full bg-custom-blue-dark justify-center"
  >
    <p
      className="text-sm font-semibold text-white"
    >
      Built by Abel
    </p>
  </footer>,
]
`;

exports[`tests for navigation bar snapshot for header 1`] = `
Array [
  <nav
    aria-controls="accordion-arrow-icon-body-1"
    className="flex justify-center w-full bg-custom-blue-dark font-lato items-center"
  >
    <p
      className="text-sm font-semibold italic text-white"
    >
      Fidelity, Bravery, Integrity
    </p>
  </nav>,
  <footer
    className="flex w-full bg-custom-blue-dark justify-center"
  >
    <p
      className="text-sm font-semibold text-white"
    >
      Built by Abel
    </p>
  </footer>,
]
`;
